### PR TITLE
#520 - Perform javascript action on click

### PR DESCRIPTION
**Pull request by remstef**
- increase column size

### CODE REVIEW

File: webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java [hunk 1]
```diff
@@ -71,7 +71,7 @@
     
     private boolean readonly = false;
     
-    @Column(nullable = true)
+    @Column(nullable = true, length = 128000)
```
Comment (reckart) [inline webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java]: This is what we normally use for such columns. Would 64kb be ok for you?

```
    @Lob
    @Column(length = 64000)
```

Comment (remstef) [inline webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java]: yes, should be sufficient :) 
what's `@Lob`?

Comment (reckart) [inline webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java]: Marks the column as a "large binary object". As far as I understand, fepending on the DB, such columns are not stored directly in the table, but rather the table contains pointers into a separate LOB storage area. Afaik that is meant to reduce the data that is kept in memory when the table are loaded/scanned.

Comment (reckart) [inline webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java]: https://docs.jboss.org/hibernate/jpa/2.1/api/javax/persistence/Lob.html

**Comment by remstef** [comment]
Alright, I changed the definition accordingly

**Pull request merged**